components/docker-up/docker-up/main.go

```diff
@@ -32,6 +35,8 @@ import (
 
 var log *logrus.Entry
 
+const DaemonArgs = "DOCKER_DAEMON_ARGS"
```

Comment:
How about this name? I feel this is simpler.
```suggestion
const DaemonArgs = "DOCKERD_ARGS"
```